nixos/modules/config/fonts/fonts.nix

```diff
@@ -2,6 +2,52 @@
 
 with lib;
 
+let
+  # A scalable variant of the X11 "core" cursor
+  #
+  # If not running a fancy desktop environment, the cursor is likely set to
+  # the default `cursor.pcf` bitmap font. This is 17px wide, so it's very
+  # small and almost invisible on 4K displays.
+  fontcursormisc_hidpi = pkgs.xorg.fontcursormisc.overrideAttrs (old:
+    let
+      # The scaling constant is 230/96: the scalable `left_ptr` glyph at
+      # about 23 points is rendered as 17px, on a 96dpi display.
+      # Note: the XLFD font size is in decipoints.
+      size = 2.39583 * config.services.xserver.dpi;
+      sizeString = builtins.head (builtins.split "\\." (toString size));
+    in
+    {
+      postInstall = ''
+        alias='cursor -xfree86-cursor-medium-r-normal--0-${sizeString}-0-0-p-0-adobe-fontspecific'
+        echo "$alias" > $out/lib/X11/fonts/Type1/fonts.alias
```

Comment:
``` suggestion
        mkdir -p $out/lib/X11/fonts/Type1
        echo "$alias" > $out/lib/X11/fonts/Type1/fonts.alias
```

Reply:
@rnhmjoj unsure if https://github.com/NixOS/nixpkgs/commit/9e8fcb0184170a97b4d0a933783bd7c83aff2ab2 was to fix this issue, but it doesn't appear that commit is in `nixpkgs-unstable` or `nixos-unstable` and this bug is preventing my system from building at the mo using those branches.

```console
error: builder for '/nix/store/8k26b5r6g0kxkd455qifiwm9b3k8yzcc-font-cursor-misc-1.0.3.drv' failed with exit code 1;
       last 10 log lines:
       >  /nix/store/1570w56jrkvr90w9x158vyb5vahnk18v-coreutils-8.32/bin/install -c -m 644 cursor.pcf.gz '/nix/store/282afm3312z6c4sj3bsfyil4jl0a80s1-font-cursor-misc-1.0.3/lib/X11/fonts/misc'
       > make  install-data-hook
       > make[2]: Entering directory '/build/font-cursor-misc-1.0.3'
       > /nix/store/yfwvq9fc7i2mbgymyzpf7az0gzv3lhx2-mkfontscale-1.2.1/bin/mkfontdir /nix/store/282afm3312z6c4sj3bsfyil4jl0a80s1-font-cursor-misc-1.0.3/lib/X11/fonts/misc
       > echo "** Warning: fonts.cache not built" ; echo "** Generate this file manually on host system using fc-cache"
       > ** Warning: fonts.cache not built
       > ** Generate this file manually on host system using fc-cache
       > make[2]: Leaving directory '/build/font-cursor-misc-1.0.3'
       > make[1]: Leaving directory '/build/font-cursor-misc-1.0.3'
       > /nix/store/vh1cpdrgrbipscys6lzr0sfcamr0r2vk-builder.sh: line 10: /nix/store/282afm3312z6c4sj3bsfyil4jl0a80s1-font-cursor-misc-1.0.3/lib/X11/fonts/Type1/fonts.alias: No such file or directory
```

Reply:
It does fix it, but the commit hasn't reached the nixos-unstable, yet.
It should be in `nixos-unstable-small`, though.

Reply:
@osslate
You can disable the offending code in the meantime `services.xserver.dpi = lib.mkForce null`.

This will bypass the evaluation.

Reply:
9e8fcb0 has propagated to nixos-unstable but I'm still seeing this error. There is no Type1/ directory in my store, so this fails.